src/tests/Rockets.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import Rockets from '../components/Rockets';
+import store from '../redux/store';
+
+describe('Rockets tests', () => {
+  test('should render', () => {
+    const rockets = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Rockets />
```

Comment:
- [ ] Your tests are not passing. Kindly try to fix this error for your tests to pass.

![image](https://user-images.githubusercontent.com/98453979/203857107-69a1b312-59d1-43c5-b69e-e46eb9430ef9.png)

Reply:
All the tests are passed now.